client/app/src/main/java/com/microsoft/research/karya/ui/registration/RegistrationViewModel.kt

```diff
@@ -59,18 +54,27 @@ class RegistrationViewModel @Inject constructor(
     val currOtpResendState: LiveData<OtpSendState>
         get() = _currOtpResendState
 
+    private val _currRegisterState = MutableLiveData<RegisterWorkerState>(RegisterWorkerState.NOT_STARTED)
+    val currRegisterState: LiveData<RegisterWorkerState>
+        get() = _currRegisterState
+
     private val _loadImageBitmap = MutableLiveData<Boolean>(false)
     val loadImageBitmap: LiveData<Boolean>
         get() = _loadImageBitmap
 
+    private val _idTokenLiveData = MutableLiveData<String>("")
+    val idTokenLiveData: LiveData<String>
+        get() = _idTokenLiveData
+
 
     var phoneNumberFragmentErrorId by Delegates.notNull<Int>()
     var otpFragmentErrorId by Delegates.notNull<Int>()
+    var selectAgeGroupFragmentErrorId by Delegates.notNull<Int>()
```

Comment:
Why not make them `lateinit`?

Reply:
Kotlin doesn't allow lateinit with primitive types